### ISSUE TITLE
Cell highlighting doesn't work on mobile (iOS) during keyboard navigation

### ISSUE THREAD

**Issue by FeralFlora** [decaf-dev/obsidian-dataloom#710]
### Describe the bug

When I navigate between cells in a loom in the mobile app, there is no visual highlighting to indicate which cell is in focus. I tested this with various themes, including the default one, all in light mode.

### Steps to reproduce

1. Click into the loom
2. Press escape to stop editing the clicked cell
3. The cell is now highlighted with the theme's accent color
4. Keyboard navigate to another cell
5. The highlighting is now gone and I have to guess which cell is in focus / selected

### Expected behavior

I would expect the cell in focus to be highlighted when I keyboard navigate to it.

### Are you using the mobile app?

Yes

### Obsidian debug info

SYSTEM INFO:
	Operating system: ios 16.6 (Apple iPad)
	Obsidian version: 1.4.6 (105)
	API version: v1.3.5
	Login status: logged in
	Catalyst license: insider
	Live preview: on
	Legacy editor: off
	Base theme: light
	Community theme: none
	Snippets enabled: 9
	Restricted mode: off
	Plugins installed: 51
	Plugins enabled: 49
		1: Kanban v1.5.3
		2: Tasks v4.6.0
		3: Dataview v0.5.56
		4: Calendar v1.5.10
		5: Homepage v3.5.1
		6: Templater v1.16.0
		7: Excalidraw v1.9.17
		8: Enhancing Mindmap v0.2.5
		9: Periodic Notes v0.0.17
		10: Projects v1.16.3
		11: Home tab v1.1.2
		12: Force note view mode v1.2.0
		13: Obsidian42 - Text Transporter v1.0.3
		14: Obsidian42 - Strange New Worlds (SNW) v1.2.2
		15: Obsidian42 - BRAT v0.7.0
		16: Buttons v0.4.19
		17: Hotkeys++ v0.2.7
		18: Outliner v4.8.0
		19: QuickAdd v1.2.1
		20: Style Settings v1.0.6
		21: Emoji Toolbar v0.4.1
		22: Natural Language Dates v0.6.1
		23: Advanced new file v1.5.1
		24: Commander v0.5.0
		25: Linter v1.19.1
		26: Auto Link Title v1.4.1
		27: txt as md v0.0.1
		28: qmd as md v0.0.2
		29: Hover Editor v0.11.13
		30: List Callouts v1.1.4
		31: LanguageTool Integration v0.3.4
		32: Dictionary v2.22.0
		33: Rollover Daily Todos v1.1.6
		34: Markdown Table Editor v0.3.1
		35: Numerals v1.2.0
		36: Tag Wrangler v0.5.11
		37: Novel word count v2.21.4
		38: Lapel v0.1.1
		39: Various Complements v8.4.1
		40: Supercharged Links v0.9.7
		41: QuickShare v1.3.1
		42: Emoji Shortcodes v2.2.0
		43: Banners v1.3.3
		44: Full Calendar v0.10.7
		45: Better Word Count v0.10.0
		46: Mermaid Tools v0.2.2
		47: Time Block Planner v0.3.0
		48: Link Exploder v0.3.0
		49: DataLoom v8.2.2

**Comment by FeralFlora** [comment]
Did you have a chance to look into this, @trey-wallis?

**Comment by trey-wallis** [comment]
I did not get a chance to look at this. Sorry for the delay.

Just to clarify, you are using a mobile app with a keyboard?

**Comment by trey-wallis** [comment]
I tried connecting my phone to my computer and controlling the table via my keyboard. The menu system seems to be working in 8.11.3. It looks like you posted this bug in 8.2.2 before the menu system update.

Could you try reproducing this?
